guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc

```diff
@@ -102,24 +102,24 @@ If you use an untypical file system (other than for example ext3, ext4, or xfs),
 . If some Pulp tasks failed due to the full disk, run them again.
 
 [id='installing-and-updating-packages-on-satellite-server']
-=== Managing Packages on the Base Operating System of {ProjectServer}
+=== Managing Packages on the Base Operating System of {Project} or {SmartProxy}
 
-To install and update packages on the {Project} base operating system, you must enter the `{foreman-maintain} packages` command.
+To install and update packages on the {Project} or {SmartProxy} base operating system, you must enter the `{foreman-maintain} packages` command.
 
-{Project} prevents users from installing and updating packages with `yum` because `yum` might also update the packages related to {Project} and result in system inconsistency.
+{Project} prevents users from installing and updating packages with `yum` because `yum` might also update the packages related to {Project} or {SmartProxy} and result in system inconsistency.
 
-IMPORTANT: The `{foreman-maintain} packages` command restarts some services on {ProjectServer} because it runs the `{foreman-installer} --upgrade` command after installing packages.
+IMPORTANT: The `{foreman-maintain} packages` command restarts some services on the operating system where you run it because it runs the `{foreman-installer} --upgrade` command after installing packages.
```

Comment:
Using `--upgrade` with `foreman-installer` is deprecated. Since Foreman 2.1 the flag is just a dummy and will be dropped entirely in the future (https://projects.theforeman.org/issues/29296).

Reply:
Nice catch, Is saying that the `satellite-maintain packages` command runs `satellite-installer` is accurate then?

Reply:
This is what the script itself tells:
WARNING: This script runs satellite-installer after the yum execution 
to ensure the Satellite is in a consistent state.
I have added a commit to fix this.

Reply:
That's indeed the best way.